lib/please_cli/please_cli/decision_task.py

```diff
@@ -407,6 +407,10 @@ def cmd(ctx,
         taskcluster_notify.irc(dict(channel='#release-services',
                                     message=f'New deployment on {channel} is about to start: https://tools.taskcluster.net/groups/{task_group_id}'))
 
+    if channel is 'production':
+        taskcluster_notify.irc(dict(channel='#ci,#moc',
```

Comment:
Are you sure you can send a message to multiple channels? i would rather call `taskcluster_notify.irc` two times. eg.:
```python
message = ("release-services team is about to release a new version of mozilla/release-services "
           "(*.mozilla-releng.net, *.moz.tools).\ Any alerts coming up soon will be best directed "
           "to #release-services IRC channel. Automated message (such as this) will be send "
           "once deployment is done. Thank you.")
for channel ['#ci', '#moc']:
    taskcluster_notify.irc(dict(channel=channel, message=message))
```